### PR TITLE
Bogdannegura experience

### PR DESCRIPTION
**Pull request by BogdanNegura**
WIP!!! :P

### CODE REVIEW

File: src/_component/experience/experience.pug [hunk 1]
```diff
@@ -0,0 +1,78 @@
+-
+    var experienceList = [
+        {
+            position: "Junior Developer",
```
[the remark this answers is not in the document]
Comment (BogdanNegura) [reply src/_component/experience/experience.pug]: Double quotes!!

[the remark this answers is not in the document]
Comment (biancalina) [reply src/_component/experience/experience.pug]: Ok!

[the remark this answers is not in the document]
Comment (BogdanNegura) [reply src/_component/experience/experience.pug]: Which one do you prefer Bianca?

[the remark this answers is not in the document]
Comment (BogdanNegura) [reply src/_component/experience/experience.pug]: Ahha 10xp each comment :))

[the remark this answers is not in the document]
Comment (biancalina) [reply src/_component/experience/experience.pug]: double quotes

[the remark this answers is not in the document]
Comment (robertoalx) [reply src/_component/experience/experience.pug]: Where is my xp?  🌊 💪

File: src/_component/experience/experience.pug [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    //-                     .experience__position=experience.position
+    //-                     .experience__company=experience.company
+    //-                 span.icon-arrow-down
+    //-         .experience-description
+    //-             p.experience-text I was spending fun time on creating amazing interfaces with React and GraphQL. I also mentored their junior and mid level staff into web development.
+
+    .space(height="30")
+    
+    .arrow-down
+        span(class=`icon-arrow-down`)
```
Comment (marianzburlea) [inline src/_component/experience/experience.pug]: double quotes

File: src/_component/experience/experience.pug [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+
+.red-box
+    .section-title Experience
+.experience
+    .space(height="30")
+    .button-wrapper
+        button.youtube-button
+            a.youtube-link(href="https://www.youtube.com/watch?v=7lgg_MSblRE")      
+                span.youtube-icon(class=`icon-youtube`)
```
Comment (marianzburlea) [inline src/_component/experience/experience.pug]: double quotes

File: src/_component/experience/experience.pug (hunk 2 above)
Comment (BogdanNegura) [reply src/_component/experience/experience.pug]: yeah, I forgot it...I encounter an error during the building the code and that's why it's
 not there :D

File: src/_component/experience/experience.pug [hunk 4]
```diff
@@ -0,0 +1,71 @@
+-
+    var experienceList = [
+        {
+            position: "Junior Developer",
+            logo: "../asset/image/arcadia.png",
+            company: "Arcadia",
+            description: "I was spending fun time on creating amazing interfaces with React and GraphQL. I also mentored their junior and mid level staff into web development."
+        },
+        {
+            position: "Rect Developer",
```
Comment (robertoalx) [inline src/_component/experience/experience.pug]: Rect? :D

File: src/_component/experience/experience.scss [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+.icon-arrow-up:hover {
+  background-color: #50d166;
+  color: white;
+}
+
+.experience-input:checked
+  + .header-experience-label
+  .header-action__arrow
+  .icon-arrow-up:before {
```
Comment (robertoalx) [inline src/_component/experience/experience.scss]: between 91 and 94 extra spaces

**Pull request merged**